### PR TITLE
De-emphasize SoD 2024 page

### PR DESCRIPTION
**Pull request by U8NWXD**

### CODE REVIEW

File: _Sidebar.md [hunk 1]
```diff
@@ -140,6 +139,6 @@
     * Google Summer of Code: [[2023|Google-Summer-of-Code-2023]], [[2022|Google-Summer-of-Code-2022]], [[2021|Google-Summer-of-Code-2021]], [[2020|Google-Summer-of-Code-2020]], [[2019|Google-Summer-of-Code-2019]], [[2018|Google-Summer-of-Code-2018]], [[2017|Google-Summer-of-Code-2017]], [[2016|Google-Summer-of-Code-2016]]
     * Hacktoberfest: [[2023|Hacktoberfest-2023]], [[2022|Hacktoberfest-2022]], [[2021|Hacktoberfest-2021]], [[2020|Hacktoberfest-2020]], [[2019|Hacktoberfest-2019]], [[2018|Hacktoberfest-2018]], [[2017|Hacktoberfest-2017]], [[2016|Hacktoberfest-2016]]
     * GHC Open Source Day: [[2019|GHC-Open-Source-Day-2019]], [[2018|GHC-Open-Source-Day]]
-    * Season of Docs: [[2021|Season-of-Docs-2021]], [[2019|Season-of-Docs-2019]]
+    * Season of Docs: [[2021|Season-of-Docs-2021]], [[2019|Season-of-Docs-2019]], [[2024|Season-of-Docs-2024]]
```
Comment (seanlip) [inline _Sidebar.md]: Should this be in order (2024 first)?

**Review by seanlip — APPROVED** [review]
Thanks @U8NWXD! Just one note.

File: _Sidebar.md (hunk 1 above)
Comment (U8NWXD) [reply _Sidebar.md]: yep, good catch!

Comment (U8NWXD) [reply _Sidebar.md]: fixed

**Pull request merged**